go/ql/src/Security/CWE-798/HardcodedCredentials.ql

```diff
@@ -60,6 +60,6 @@ where
   message = "Hard-coded private key."
   or
   HardcodedCredentials::Flow::flow(source, sink) and
-  type = SensitiveExpr::password() and
-  message = "Hard-coded credential."
+  type = SensitiveExpr::secret() and
+  message = "Hard-coded $@."
```

Comment:
I also changed this so that the alert contains a link to the source (i.e. the hard-coded string).